src/components/container/theme.css

```diff
@@ -0,0 +1,19 @@
+@import '@teamleader/ui-utilities';
+
+.container {
+  min-width: 990px;
+  padding-left: var(--spacer-big);
+  padding-right: var(--spacer-big);
+}
+
+.is-fixed {
+  margin: 0 auto;
+  max-width: 1080px;
```

Comment:
Why max-width? Now it will still be kind of fluid, no?

Reply:
You're right @lowiebenoot, `max-width` should equal to `min-width` when the container is fixed. 

Also, the padding should not increase on larger screens because margin: 0 auto; should take care of this. And I also changed the `box-sizing` of the container to be sure the width is always correct regardless of the paddings on each side.